### DISCUSSION TITLE
All Supported Games & Launchers

### DISCUSSION THREAD

**Discussion by LegItMate** [Lazap-Development/lazap#53] [General]
# Supported Games & Launchers :hammer_and_pick: 
Lazap should and must have as many games and launchers supported as possible. And for that we will track the games/launchers supported and the games/launchers that should be supported in the future! Including community suggestions and feedback on what else should be changed. 

### List of Supported Games/Launchers<a id="supported-games">
Here are all the games/launchers supported as of October 2022.
- Epic Games

- Lunar Client
- Lutris
- Minecraft Launcher
- Riot Games
- Steam
- Uplay
- Custom Games
- Rockstar Games
- Xbox Games

### List of Planned Support for games/launchers<a id="planned-games">
Some future games/launchers planned to be supported are:
- *some suggestion here*

<br>

Feel free to request addition of support for your favourite game/launcher to Lazap, that is if it is not listed here. We will try to keep this list updated as much as possible!

**Comment by mekathegapple** [comment]
What about Origin? Are you planning to add it?

**Reply by LegItMate** [reply]
Not really... it was in some sort of list but right now it is not planned in our upcoming features.

Though it seems pretty easy to support Origin Games so it might be added soon!
